### DISCUSSION TITLE
v1.109.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#11187] [Announcements]
# v1.109.0

> [!IMPORTANT] 
> Read our announcement about supporting Immich by buying a license [here](https://github.com/immich-app/immich/discussions/11186)

## Highlights

Welcome to version `v1.109.0` of Immich. This release introduces an additional way for you to support Immich financially as well as bug fixes for various issues. Some of the highlights in this release include:

- Immich licenses (and https://buy.immich.app)
- "My Immich" url forwarder (https://my.immich.app)
- Notable fix: Generate thumbnail for iOS18 HEIC
- Notable fix: Generate Motion Photos for Pixel 6, 7, and 8

### Immich licenses

With the [announcement of Immich's licensing](https://github.com/immich-app/immich/discussions/11186), you can now purchase a license to support Immich and open-source software.

In the web app, there is an information box that shows if you are a licensed or unlicensed user. You can click on the box to navigate to the purchase page or to enter the license key directly.

| Unlicensed | Licensed |
| ----------- | --------- |
|![Unlicensed](https://github.com/user-attachments/assets/6a1094c1-518b-4369-9889-8dacc88c8297) | ![Licensed](https://github.com/user-attachments/assets/782959b0-23c2-46f4-8aca-6338081de2e5) |

### "My Immich" url forwarder (https://my.immich.app)

To help with future support, we are happy to introduce a "proxy" mechanism to help us navigate you to the exact location on your instance without knowing your instance URL. 

The website prompts for your instance URL on the first visit and stores it client-side. It automatically redirects on subsequent visits, making it convenient to link to specific pages in self-hosted instances. 

Below are some of the example use cases:

- Linking directly to relevant pages from the documentation site.
- Sharing the location of a particular setting in a support thread.
- Automatically activate your instance after you buy a license.

![image](https://github.com/user-attachments/assets/cb740b11-51ec-44e1-849d-6396ddfc7244)

## Support Immich

<p align="center">
<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExbjY2eWc5Y2F0ZW56MmR4aWE0dDhzZXlidXRmYWZyajl1bWZidXZpcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/87CKDqErVfMqY/giphy.gif" width="450" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich by purchasing a license at [https://buy.immich.app](https://buy.immich.app).

Cheers! 🍻

Immich team

### 🗄️ Server
* feat(server): userinfo signing by @jrasm91 in https://github.com/immich-app/immich/pull/10756
* feat(server): conditionally run facial recognition nightly by @mertalev in https://github.com/immich-app/immich/pull/11080
* fix: #11131 storage migration not moving archived files after template change by @zackpollard in https://github.com/immich-app/immich/pull/11139
* fix: downgrade exiftool-vendored to fix motion photo extraction by @zackpollard in https://github.com/immich-app/immich/pull/11145
* fix(server): delete large album by @jrasm91 in https://github.com/immich-app/immich/pull/11042
* feat(server): country geocoding for remote locations by @pokjay in https://github.com/immich-app/immich/pull/10950
* feat(web): license UI by @alextran1502 in https://github.com/immich-app/immich/pull/11182
### 📱 Mobile
* fix(mobile): fix database out of size by @alextran1502 in https://github.com/immich-app/immich/pull/11065
### 🖥️ Web
* fix(web): prevent text wrapping for some languages by @michelheusschen in https://github.com/immich-app/immich/pull/11009
* feat(web): change mdiClose to mdiInfinity for User Management - quota by @MickLesk in https://github.com/immich-app/immich/pull/11015
* chore(web): update translations by @weblate in https://github.com/immich-app/immich/pull/11007
* feat(web): Rename the Menu Entry "admin.map_settings" to "admin.map_gps_settings" by @MickLesk in https://github.com/immich-app/immich/pull/11033
* chore(web): use "immich-form-label" class for combobox label by @flesser in https://github.com/immich-app/immich/pull/10254
* chore: clean up change date form spacing by @jrasm91 in https://github.com/immich-app/immich/pull/11037
* refactor(web): remove buffer package by @michelheusschen in https://github.com/immich-app/immich/pull/11058
* chore(web): update translations by @weblate in https://github.com/immich-app/immich/pull/11038
* fix(web): byte units enum by @michelheusschen in https://github.com/immich-app/immich/pull/11161
* fix(web): more translations by @waclaw66 in https://github.com/immich-app/immich/pull/11167
### 📓 Documentation
* fix(doc): Fix broken link in Cursed Knowledge doc by @AverageHelper in https://github.com/immich-app/immich/pull/11008
* docs: fix images by @jrasm91 in https://github.com/immich-app/immich/pull/11012
* chore(docs): correct susceptible spelling by @adieltan in https://github.com/immich-app/immich/pull/11029
* fix(docs): minor grammar fix in external-library.md by @Rajdave69 in https://github.com/immich-app/immich/pull/11073
* docs: cleanup external libraries by @mmomjian in https://github.com/immich-app/immich/pull/11099
* docs: facial recognition and general clean-up by @mertalev in https://github.com/immich-app/immich/pull/11106
* docs: install script note by @mmomjian in https://github.com/immich-app/immich/pull/11122
* chore(ml): set higher worker timeout for openvino by @mertalev in https://github.com/immich-app/immich/pull/11174
### 🔨 Maintenance
* chore(deps): update dependency rimraf to v6 by @renovate in https://github.com/immich-app/immich/pull/11079
* chore(deps): update base-image to v20240713 (major) by @renovate in https://github.com/immich-app/immich/pull/11066
* chore(deps): update typescript-projects by @renovate in https://github.com/immich-app/immich/pull/11129
* chore(deps): update terraform cloudflare to v4.37.0 by @renovate in https://github.com/immich-app/immich/pull/11132
* chore(deps): update base-image to v20240716 (major) by @renovate in https://github.com/immich-app/immich/pull/11144
* chore(deps): update base-image to v20240717 (major) by @renovate in https://github.com/immich-app/immich/pull/11172
### Other changes
* chore(mobile): post release task by @alextran1502 in https://github.com/immich-app/immich/pull/11006
* chore(docs): update German Readme (README_de_DE.md) by @MickLesk in https://github.com/immich-app/immich/pull/11035
* chore: Update feature-request.yaml by @bo0tzz in https://github.com/immich-app/immich/pull/11150

## New Contributors
* @AverageHelper made their first contribution in https://github.com/immich-app/immich/pull/11008
* @MickLesk made their first contribution in https://github.com/immich-app/immich/pull/11015
* @adieltan made their first contribution in https://github.com/immich-app/immich/pull/11029
* @flesser made their first contribution in https://github.com/immich-app/immich/pull/10254
* @Rajdave69 made their first contribution in https://github.com/immich-app/immich/pull/11073
* @pokjay made their first contribution in https://github.com/immich-app/immich/pull/10950

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.108.0...v1.109.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.109.0'>v1.109.0</a>.</em>

**Comment by athornfam2** [comment]
Well that didn't take too long to ask for money to support immich via a license key

**Reply by bo0tzz** [reply]
The [Futo announcement](https://immich.app/blog/2024/immich-core-team-goes-fulltime) a few months ago already made clear that this was the plan.

**Reply by athornfam2** [reply]
Obviously the decision is in the hands of "Futo". However, I believe the community would have been more open to a support model similar to Bookstack. Community support via a forum is free (discord in this case) while other support options via ticket are paid. In addition, people probably would love to have a hosted platform through Immich. Subscribe at a lower fee than google photos or different tiered storage options. I'm sure a lot of people would like to purchase a hosted platform and not worry about anything more than managing it through an admin portal

**Reply by maxshcherbina** [reply]
I'm confused, there's nothing locked behind a license, right? Seems to me like just a nice way to support the devs? Or are Futo products known to have features locked behind licenses?

**Reply by ChrislyBear-GH** [reply]
This is just great...
Sure, right now being a part of FUTO might look good. But wait until FUTO gets swallowed by a larger corporation.
Prices will increase, lifetime licenses won't be "lifetime" anymore. Features will be paywalled. I've seen it too often, it's just a matter of time.

Don't get me wrong, I'll probably buy a lifetime server license, because I want to believe that this time it'll be different and this project will be the exception. One can hope...

**Reply by bo0tzz** [reply]
Nothing is locked behind the license, and nothing ever will be.

> But wait until FUTO gets swallowed by a larger corporation.
Prices will increase, lifetime licenses won't be "lifetime" anymore. Features will be paywalled.

If this happens, we quit and fork Immich. If we don't, someone else can.

**Reply by ChrislyBear-GH** [reply]
> Nothing is locked behind the license, and nothing ever will be.
> 
> > But wait until FUTO gets swallowed by a larger corporation.
> > Prices will increase, lifetime licenses won't be "lifetime" anymore. Features will be paywalled.
> 
> If this happens, we quit and fork Immich. If we don't, someone else can.

My fear isn't, that you and the current team will "sell out"! I hope it didn't sound like this, and if it did, I'm sorry.
But things can happen, companies might merge, intellectual property transfers etc. and suddenly everything is completely different...

Nevertheless, your assurances are very appreciated and they give me hope, that this project might really be the exception! :)

**Reply by alextran1502** [reply]
You can watch the video we talked about all your concerns here https://www.youtube.com/watch?v=cwz2iZwYpgg

**Reply by alextran1502** [reply]
> Obviously the decision is in the hands of "Futo". However, I believe the community would have been more open to a support model similar to Bookstack. Community support via a forum is free (discord in this case) while other support options via ticket are paid. In addition, people probably would love to have a hosted platform through Immich. Subscribe at a lower fee than google photos or different tiered storage options. I'm sure a lot of people would like to purchase a hosted platform and not worry about anything more than managing it through an admin portal

Just to be cleared, all of the idea about licensing, pricing are coming from us, nothing is being pushed from FUTO side

**Reply by BoBBer446** [reply]
> Nothing is locked behind the license, and nothing ever will be.
> 
> > But wait until FUTO gets swallowed by a larger corporation.
> > Prices will increase, lifetime licenses won't be "lifetime" anymore. Features will be paywalled.
> 
> If this happens, we quit and fork Immich. If we don't, someone else can.

Oh, what a shame. How often have you heard "it will always be free" and a few weeks, months or a year or two later the first little things start to happen. Then there's this and that for licensees and then it gets more and more. 

Sorry, but I think I'm done with that. It's extremely unfortunate that this kind of monetization is happening now. It's sad.

Reminds me of WhatsApp and Meta...how often was this and that promised and sworn. It's all smoke and mirrors in IT. 

Immich and the community are being sold. Too bad.
You know it, i know it...all know it :/

**Reply by MickLesk** [reply]
@Immich-Team @alextran1502 @bo0tzz @danieldietzler @martabal and co

Perhaps the license is simply the wrong model or the wrong spelling. 

As a German, I always break into a cold sweat when I read something about a license. 
It makes much more sense to define it as “supporter” or “sponsorship”, doesn't it? 

The word license always has something binding about it, there's always a “payment model” in the back of your mind. 
I had already thought about translating it to “Unterstützen” in the Weblate (since I have done most of the translation for German there), but I don't want to do that without talking about it. 

Daniel will understand it most of all, as he is also a stingy German :-D


**Edit:** Forgot it, ive seen this thread: https://github.com/immich-app/immich/discussions/11313 👍🏻

**Reply by danieldietzler** [reply]
> @Immich-Team @alextran1502 @bo0tzz @danieldietzler @martabal and co
> 
> Perhaps the license is simply the wrong model or the wrong spelling. 
> 
> As a German, I always break into a cold sweat when I read something about a license. 
> It makes much more sense to define it as “supporter” or “sponsorship”, doesn't it? 
> 
> The word license always has something binding about it, there's always a “payment model” in the back of your mind. 
> I had already thought about translating it to “Unterstützen” in the Weblate (since I have done most of the translation for German there), but I don't want to do that without talking about it. 
> 
> Daniel will understand it most of all, as he is also a stingy German :-D
> 

@MickLesk you should also read the follow-up post: https://github.com/immich-app/immich/discussions/11313

**Reply by MickLesk** [reply]
@danieldietzler ive seen it some minutes ago and edit my post 👍🏻

**Comment by LucaDev** [comment]
Please have a look at the discussion in https://github.com/immich-app/immich/pull/11182 before commenting on the license. I think as long as there is only a small banner - it’s completely fine.

**Comment by andrewdunndev** [comment]
For the motion photos fix, did we need to run a specific job to be able to view those now?

**Reply by jrasm91** [reply]
Metadata extraction on the affected assets should fix them.

**Comment by nicomerl** [comment]
I was a Github Donator in the past and while i absolutely hate that today everything needs an Abo and monthly payment, i do Unterstand that espacially Software ist developed and supported over years so regular income is needed.

i made some Bad experiences with „lifetime“ Licenses After the Software vendor realised that one time payment doesnt provide enough Money forever.

Nevertheless: i love Immich, the Team around it and the vision that my data stays mine. I‘ll buy a lifetime license. But to say it more or less with Erons words from the Q&A: Software developers in the past did everything to lose Trust from their customers.

**Reply by stephen304** [reply]
I agree that "lifetime" licenses can be dubious, but in this case I don't think there can be any issue respecting the license because it doesn't actually unlock any features. That seems to be how Futo does things since I have the grayjay and keyboard/voice input licenses which also are just be donations and don't unlock anything, so there isn't really any ongoing cost to them.

That said I do agree that calling it a lifetime license is confusing until you actually read the FAQs (or unless you're already familiar with the "donation licenses" in other futo apps), and I think it should be called buying a "Supporter Badge" instead with the in-app badge saying something like "Supporter".

**Comment by 0verEngineer** [comment]
"License" and "Free trial" are definitely the wrong words for this stuff, Immich is already licensed to me through the AGPL license...

**Comment by BoBBer446** [comment]
Nothing in the world is free, not even death. 
Anyone who believes otherwise is a fool.
The crucial point is that imich developers are now paid full time and the one who pays the bill wants something in return. Sooner or later. 

Nobody can seriously believe that sooner or later something will not be expected in return and not demanded later.

I love immich and the dedication of everyone involved and I love the community. It will probably take a while, but this first stone will not be the last. We know it. Even if not everyone wants to admit it. I feel sorry for the developers who have taken such loving care of the project so far.

I don't want to rush, I don't want to talk badly. I just wanted to express my opinion. I'm not going to comment any further on the subject and hope that I'll soon look like an idiot for being wrong. 

Aidos my friends <3

**Comment by yznts** [comment]
I don't mind supporting such a great project, but talking about support in the way of "License" without even having a stable releases seems strange to me.

**Comment by ThatCoffeeGuy** [comment]
I'd honestly rather see the donation brought back in some form. I love this software. I want to (and have before) support it by donations, but $25 or $99 is simply too much. 

So Instead of being able to pay $5-$10 every six months, I now only have the option of $25 or $99. This makes me feel bad for not being able to donate, and it means you don't receive my donation either. I don't know how many people are in this situation, but if many had made smaller donations, this change could result in a lower total amount of donations in the long run.

**Comment by jagjordi** [comment]
How does this leave people that support by other means? I have been donating through github for a while now, I probably have donated more than the cost of a "1 user license". Will you guys give licenses to people that donate through other means?

**Reply by zackpollard** [reply]
We cancelled all donations to the GitHub org when we got taken on full time by Futo. If you're still donating it's likely to Alex directly who sent out a mail to everyone inviting them to cancel if they no longer want to support. We are going to make a method to claim licenses based on the amount you donated in the past probably within the first couple days this coming week. This will be a one-time thing though based on the current donations from all-time to Alex or the org. This won't continue going forward if you choose to continue donating to one of the individual contributors of the project 🙂 Thanks for your support!

**Comment by jeroenvaes** [comment]
Hey Immich-team! Just wanted to say thank you for this amazing piece of software. I'll be buying a license/activation key/whatever shortly.

Whilst I'm not sure about the sustainability of the lifetime license, I applaud you for trying something different. It's refreshing, but also quite funny you are getting slack for this because I'm old enough to remember times where you could just...buy software for a flat fee and be done with it. 😂

And for people complaining, please remember that this team don't owe you anything. Not even if you donated in the past. They are providing this piece of software and you can use it. Or not. And now you can even pay for it. Sure they can yank the lifetime license, or throw features behind a paywall, but that is currently not the case and so far they are doing everything in a very transparent manner. So let's not get out the pitchforks just yet and see how this plays out.
